src/Seld/JsonLint/JsonParser.php

```diff
@@ -450,6 +450,10 @@ private function performAction(stdClass $yyval, $yytext, $yyleng, $yylineno, $yy
 
     private function stringInterpolation($match)
     {
+        if (strpos($match[0], '\u') === 0) {
```

Comment:
I would move that inside the default, to avoid doing that string scanning for all cases matched by the `switch` case

Reply:
good point 👍